### PR TITLE
Update xrootd to version 5.7.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by iarspider** [comment]
please test

**Comment by smuzaffar** [comment]
@iarspider , does `isal` has shared libs? If yes then I would suggest to add its tools file too

**Comment by iarspider** [comment]
nasm is only needed for x86_64: https://github.com/intel/isa-l/blob/master/README.md?plain=1#L41-L43

**Comment by iarspider** [comment]
please test

File: scram-tools.file/tools/isal/isal.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<tool name="isal" version="@TOOL_VERSION@">
+  <info url="https://github.com/intel/isa-l/wiki"/>
+  <lib name="ISAL"/>
```
Comment (smuzaffar) [inline scram-tools.file/tools/isal/isal.xml]: @iarspider , it is generating libISAL.so?

**Comment by iarspider** [comment]
please test

**Comment by smuzaffar** [comment]
please test

lets use the latest IB

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**